pkg/logproto/logproto.proto

```diff
@@ -119,6 +119,7 @@ message LabelRequest {
     (gogoproto.stdtime) = true,
     (gogoproto.nullable) = true
   ];
+  string query = 5;
```

Comment:
So we use this proto somewhere internally

```protobuf

enum LabelMatcherType {
  LABEL_MATCHER_TYPE_UNSPECIFIED  = 0;
  LABEL_MATCHER_TYPE_EQ  = 1;
  LABEL_MATCHER_TYPE_NEQ = 2;
  LABEL_MATCHER_TYPE_RE  = 3;
  LABEL_MATCHER_TYPE_NRE = 4;
}

message LabelMatcher {
  LabelMatcherType type    = 1;
  string name  = 2;
  string value = 3;
};
```

And convert it to `labels.Matcher`. I'm not sure this is worthwhile.

Reply:
I think you'd have to add another call which might be too much work.